run.py

```diff
@@ -157,17 +152,17 @@ def reindex(since=None, batch_size=None, force_indexing=False, update_solr=True,
                     batch.append(rec.bibcode)
                     tasks.task_index_records.delay(batch, force=force_indexing, update_solr=update_solr,
                                                    update_metrics=update_metrics, update_links=update_links,
-                                                   ignore_checksums=ignore_checksums)
+                                                   ignore_checksums=ignore_checksums, priority=priority)
```

Comment:
There are three at least three other calls to `task_index_records` where we are not propagating the new priority argument (see function `reindex_failed` and `diagnose`). Could you add the priority parameter please?

Reply:
Good catch, perhaps a new unit test is called for.